components/list/list-controls.js

```diff
@@ -19,31 +20,48 @@ export class ListControls extends SelectionControls {
 				--d2l-selection-controls-padding: var(--d2l-list-controls-padding, 18px);
 				z-index: 6; /* must be greater than d2l-list-item-active-border */
 			}
+			:host([child-has-color]),
+			:host([_extend-separator]) {
```

Comment:
Does this need to be applied to the host, or could it be applied to the main container that `SelectionControls` renders. We didn't really want to sprout private properties on the host if we could help it, which is part of the reason why `_getSelectionControlsContainerClasses` was added - it enables `List`/`TableControls` to specify CSS classes to be rendered on the main internal container element even though they aren't doing any of the actual HTML rendering.

Reply:
I'll fiddle with it. There's just some weirdness in the bottom corners (I think it's to do with the drop shadow) on hover when the margin is on the section vs the host:
<img width="386" alt="Screenshot 2023-09-01 at 8 47 03 AM" src="https://github.com/BrightspaceUI/core/assets/6804600/723d5f10-27c5-4ce2-8c8c-f86d920f5aef">